packages/integrations/cloudflare/src/entrypoints/preview.ts

```diff
@@ -11,40 +11,30 @@ import type * as http from 'node:http';
 import colors from 'piccolore';
 import { performance } from 'node:perf_hooks';
 import { hasWranglerConfig, defaultCloudflareConfig } from '../wrangler.js';
-import { copyFileSync, existsSync } from 'node:fs';
 
 const createPreviewServer: CreatePreviewServer = async ({
 	logger,
 	base,
-	outDir,
+	server,
 	headers,
 	port,
 	host,
-	createCodegenDir,
 	root,
 }) => {
 	const startServerTime = performance.now();
 	let previewServer: VitePreviewServer;
 	let cfPluginConfig: PluginConfig = { viteEnvironment: { name: 'ssr' } };
-	if (!hasWranglerConfig(outDir)) {
+	if (!hasWranglerConfig(server)) {
```

Comment:
This used to always be false because the wrangler config is in `dist/_wrangler.js`, not `dist`